src/test/java/org/sasanlabs/service/vulnerability/commandInjection/CommandInjectionTest.java

```diff
@@ -0,0 +1,119 @@
+package org.sasanlabs.service.vulnerability.commandInjection;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.verify;
+import static org.sasanlabs.vulnerability.utils.Constants.LOCALHOST;
+
+import java.io.IOException;
+import java.net.URI;
+import org.apache.logging.log4j.util.Strings;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.sasanlabs.service.exception.ServiceApplicationException;
+import org.sasanlabs.service.vulnerability.bean.GenericVulnerabilityResponseBean;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.RequestEntity;
+import org.springframework.http.ResponseEntity;
+
+/*
+ * @author Feng-Yuan Yang coolesaltw@hotmail.com
+ */
+public class CommandInjectionTest {
+
+    private static CommandInjection commandInjection;
```

Comment:
No need for this variable to be static. Please remove static from this declaration.